### DISCUSSION TITLE
link to file resolves wrong in markdown preview

### DISCUSSION THREAD

**Discussion by mgmegive** [marp-team/marp#461] [Q&A] [answered]
Hello, I've discovered what I believe to be a bug but wanted to confirm here first (per form) it was in fact a bug and not a user error.

I am developing my second slide deck using Marp. In said deck, I'm linking to another markdown file. However, in the built in Markdown Preview within VS Code, the link opens my web browser instead of opening said markdown file. If I disable marp, the link resolves correctly through the preview. Also, the links in the PDF export resolve correctly.

Below is a small sample that explains demonstrates syntax that is exhibiting this behavior. I'm curious if someone else can reproduce this and confirm whether or not this is a bug.

```
---
marp: true
---

# First Slide

- words
- [link to url](https://github.com/)
  - resolves correctly
- [link to file](img/temp.md)
  - resolves to `https://file+.vscode-resource.vscode-cdn.net/c%3A<PROPER-PATH>/img/temp.md`
```

**Comment by yhatt** [comment, marked as answer]
That will fix by https://github.com/marp-team/marp-vscode/pull/428.

**Reply by yhatt** [reply]
**UPDATE**: It should have been fixed in the latest [Marp for VS Code v2.7.0](https://github.com/marp-team/marp-vscode/releases/tag/v2.7.0).
